lib/error-handler.js

```diff
@@ -95,6 +95,7 @@ function defaultErrorHandler (error, request, reply) {
 function fallbackErrorHandler (error, reply, cb) {
   const res = reply.raw
   const statusCode = reply.statusCode
+  reply[kReplyHeaders]['content-type'] ??= 'application/json; charset=utf-8'
```

Comment:
nullish coalescing assignment is not supported in node 14

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment#browser_compatibility

```suggestion
  reply[kReplyHeaders]['content-type'] = reply[kReplyHeaders]['content-type'] ?? 'application/json; charset=utf-8'
```